Source/WebCore/page/scrolling/mac/ScrollerPairMac.mm

```diff
@@ -147,9 +147,14 @@ - (void)scrollerImpPair:(NSScrollerImpPair *)scrollerImpPair updateScrollerStyle
 {
     [m_scrollerImpPairDelegate invalidate];
     [m_scrollerImpPair setDelegate:nil];
+    
+    m_verticalScroller.detach();
+    m_horizontalScroller.detach();
 
     ensureOnMainThread([scrollerImpPair = std::exchange(m_scrollerImpPair, nil)] {
     });
+    releaseReferencesToScrollerImpsOnTheMainThread();
```

Comment:
Seems weird to have two separate bounces to the main thread here. Can we combine them?

Reply:
Fixed.